### DISCUSSION TITLE
2021년 회고 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#25] [Blog Comments]
# 2021년 회고 - peterkimzz

다들 새해 다짐 잘 실천하고 계신가요? 전 아직 다짐을 못했습니다. 약간의 업무 슬럼프가 오는 것 같아서 뭘 해야할지 아직 계획이 없네요. 날 좀 풀리면 운동부터 다시 시작해야할 것 같습니다. 코로나가 얼른 끝났으면 좋겠습니다.
개인 블로그
이 블로그를 운영한지 1년 

https://www.peterkimzz.com/1-year-review-of-blog-operation-and-2021/

**Comment by nemo4** [comment]
흘러흘러 회고록까지 둘러보게되었네요. 좋은글 잘보고갑니다.

**Reply by peterkimzz** [reply]
감사합니다!
